### PR TITLE
Formatting l10n module

### PR DESCRIPTION
**Pull request by g1itch**
Another portion of formatting for the code grooming.

### CODE REVIEW

File: src/l10n.py [hunk 1]
```diff
@@ -40,55 +50,51 @@
...
+        time.strftime(
+            time_format, (0, 0, 0, 11, 0, 0, 0, 0, 0)).decode(encoding)
+        time.strftime(
+            time_format, (0, 0, 0, 13, 0, 0, 0, 0, 0)).decode(encoding)
         # Check DST
-        unicode(time.strftime(time_format, (0, 0, 0, 0, 0, 0, 0, 0, 1)), encoding)
-    except:
+        time.strftime(
+            time_format, (0, 0, 0, 0, 0, 0, 0, 0, 1)).decode(encoding)
+    except Exception:  # TODO: write tests and determine exception types
```
Comment (g1itch) [inline src/l10n.py:80]: should be `UnicodeDecodeError` but again, I cannot write a test

File: src/l10n.py [hunk 2]
```diff
@@ -40,55 +50,51 @@
...
     # it looks like some stuff isn't initialised yet when this is called the
     # first time and its init gets the locale settings from the environment
     os.environ["LC_ALL"] = newlocale
 
 
-def formatTimestamp(timestamp=None, as_unicode=True):
+def formatTimestamp(timestamp=None):
     """Return a formatted timestamp"""
     # For some reason some timestamps are strings so we need to sanitize.
     if timestamp is not None and not isinstance(timestamp, int):
```
Comment (g1itch) [inline src/l10n.py:100]: Here is probably also `ValueError`

**Comment by g1itch** [comment]
More I read this module, a more code seems redundant and written just in case ):

This will be squashed into 2 commits if there will be no objections. The `range` from `six.moves` seems excessive here (#1837); again, nobody reads any PR ):

**Review by PeterSurda — APPROVED** [review]
Looks ok, I hope to have windows workers on buildbot soon so that the tests can run there too.

**Pull request merged**